views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
```

Comment:
This should be enumerated using the `for` template helper, and embedded within a `ul` so that each person attending appears as a `li`.